lms/djangoapps/course_api/blocks/transformers/navigation.py

```diff
@@ -1,7 +1,7 @@
 """
 TODO
```

Comment:
Since we're already in this file, could we update the module docstring to be something other than this line?